publication-model/src/main/resources/publication-ontology.ttl

```diff
@@ -960,6 +960,82 @@ nva:DegreeLicentiate a rdfs:Class ;
   rdfs:isDefinedBy nva: ;
   rdfs:subClassOf nva:Degree .
 
+nva:AllowedFileOperation a rdfs:Class ;
+  rdfs:comment "A thing that indicates what you can do with a file" ;
+  rdfs:isDefinedBy nva: .
+
+nva:AllowedPublicationOperation a rdfs:Class ;
+  rdfs:comment "A thing that indicates what you can do with a publication" ;
+  rdfs:isDefinedBy nva: .
+
+nva:read_metadata a nva:AllowedFileOperation ;
+  rdfs:comment "Indicates read access is allowed" ;
+  rdfs:isDefinedBy nva: .
+
+nva:write_metadata a nva:AllowedFileOperation ;
+  rdfs:comment "Indicates write access is allowed" ;
+  rdfs:isDefinedBy nva: .
+
+nva:delete a nva:AllowedFileOperation ;
+  rdfs:comment "Indicates delete access is allowed" ;
+  rdfs:isDefinedBy nva: .
+
+nva:download a nva:AllowedFileOperation ;
+  rdfs:comment "Indicates download access is allowed" ;
+  rdfs:isDefinedBy nva: .
+
+nva:update a nva:AllowedPublicationOperation ;
+  rdfs:comment "Indicates update access is allowed" ;
+  rdfs:isDefinedBy nva: .
+
+nva:update_files a nva:AllowedPublicationOperation ;
+  rdfs:comment "Indicates update including files access is allowed" ;
+  rdfs:isDefinedBy nva: .
+
+nva:read_hidden_files a nva:AllowedPublicationOperation ;
+  rdfs:comment "Indicates read hidden files access is allowed" ;
+  rdfs:isDefinedBy nva: .
+
+nva:unpublish a nva:AllowedPublicationOperation ;
+  rdfs:comment "Indicates unpublish access is allowed" ;
+  rdfs:isDefinedBy nva: .
+
+nva:republish a nva:AllowedPublicationOperation ;
+  rdfs:comment "Indicates republish access is allowed" ;
+  rdfs:isDefinedBy nva: .
+
+nva:delete_pub a nva:AllowedPublicationOperation ;
+  rdfs:comment "Indicates delete publication access is allowed" ;
+  rdfs:isDefinedBy nva: .
+
+nva:terminate a nva:AllowedPublicationOperation ;
+  rdfs:comment "Indicates terminate access is allowed" ;
```

Comment:
Don't know what this means at all…terminate?

Reply:
delete is a "full" delete, terminate removed files and some metadata (or should at least)

Reply:
Indicates the user may remove files and metadata from the publication by terminating the publication